### PR TITLE
Add black

### PR DESCRIPTION
**Pull request by bruziev**
## What do these changes do?

- Add black
- Make first formatting generated project
- Add rule for Makefile

## Related issue number

#70 

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is `<Name> <Surname>`.
  * Please keep alphabetical order, the file is sorted by names. 
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: create_aio_app/template/{{cookiecutter.project_name}}/Makefile [hunk 1]
```diff
@@ -27,10 +27,14 @@ _doc:
 
 # Linters & tests
 
+format:
+	@docker-compose run --rm $(PROJECT_NAME)_app black $(PROJECT_NAME) --exclude $(PROJECT_NAME)/migrations -l 79
+
 mypy:
 	@docker-compose run --rm $(PROJECT_NAME)_app mypy $(PROJECT_NAME)
 
 lint:
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/Makefile]: u can use your format command here.
```
lint: format
   @docker-compose run --rm $(PROJECT_NAME)_app flake8 $(PROJECT_NAME)
```

File: create_aio_app/main.py [hunk 2]
```diff
@@ -2,10 +2,7 @@
 from pathlib import Path
 
 import click
-from cookiecutter.exceptions import (
-    FailedHookException,
-    OutputDirExistsException
-)
+from cookiecutter.exceptions import FailedHookException, OutputDirExistsException
```
Comment (Arfey) [inline create_aio_app/main.py]: i prefer use black only for created project not for package.

File: create_aio_app/template/{{cookiecutter.project_name}}/docs/source/pages/commands.rst [hunk 3]
```diff
@@ -29,7 +29,8 @@ Testing
 
    ``make test``, To start the ``pytest`` with docker
    ``make mypy``, To run ``mypy`` for type checking
-   ``make lint``, To run flake8 (The all settings connected with a ``flake8`` you can customize in ``.flake8``)
+   ``make format``, To run ``black`` for auto formatting code
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/docs/source/pages/commands.rst]: `make black` looks more organic

File: create_aio_app/template/{{cookiecutter.project_name}}/Makefile [hunk 4]
```diff
@@ -27,10 +27,14 @@ _doc:
...
+	@docker-compose run --rm $(PROJECT_NAME)_app black $(PROJECT_NAME) --exclude $(PROJECT_NAME)/migrations --check -l 79
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/Makefile]: don't concat this commands

**Comment by bruziev** [comment]
> First, thanks for your MR 😀
> Black should be a optional useful tool. Therefore, you just need to add the black to description + docs + makefile. The project format should remain unchanged.

Done

File: create_aio_app/main.py (hunk 2 above)
Comment (bruziev) [reply create_aio_app/main.py]: reverted

File: create_aio_app/template/{{cookiecutter.project_name}}/Makefile (hunk 4 above)
Comment (bruziev) [reply create_aio_app/template/{{cookiecutter.project_name}}/Makefile]: remove from lint 'cause formatting is optional

**Comment by Arfey** [comment]
@bruziev right. But you forgot about very important part. Add yourself to the `CONTRIBUTORS.txt` 😀

**Comment by Arfey** [comment]
thx!

**Pull request merged**